build.gradle

```diff
@@ -39,7 +39,9 @@ dependencies {
 
     compile("uk.ac.ebi.subs:subs-processing-model:0.1.1-SNAPSHOT")
     compile("uk.ac.ebi.subs:subs-data-model:0.1.0-SNAPSHOT")
-    compile("uk.ac.ebi.subs:subs-messaging:0.1.1-SNAPSHOT")
+    compile("uk.ac.ebi.subs:subs-messaging:0.2.0-SNAPSHOT") {
+        changing = true
```

Comment:
Since we are versioning, why do we need the `changing = true` flag ?

Reply:
Agreed. Removed.